pkgs/development/ocaml-modules/frontc/default.nix

```diff
@@ -0,0 +1,32 @@
+{lib, buildOcaml, fetchurl}:
+
+buildOcaml rec {
+  name = "Frontc";
+  version = "3.4";
+
+  src = fetchurl {
+    url = "http://www.irit.fr/recherches/ARCHI/MARCH/frontc/${name}-${version}.tgz";
+    sha256 = "16dz153s92dgbw1rrfwbhscy73did87kfmjwyh3qpvs748h1sc4g";
+  };
+
+  meta = with lib; {
+    homepage = https://www.irit.fr/recherches/ARCHI/MARCH/rubrique.php3?id_rubrique=61;
+    description = "C Parsing Library";
+    license = licenses.lgpl21;
+    maintainers = [ maintainers.maurer ];
+  };
+
+  meta_file = fetchurl {
+    url = https://raw.githubusercontent.com/ocaml/opam-repository/0f0e610f6499bdf0151e4170411b4f05e4d076d4/packages/FrontC/FrontC.3.4/files/META;
+    sha256 = "1flhvwr01crn7d094kby0418s1m4198np85ymjp3b4maz0n7m2mx";
+  };
+
+  makeFlags = "PREFIX=$(out) OCAML_SITE=$(OCAMLFIND_DESTDIR)";
+
+  postInstall  = ''
+     cp ${meta_file} META
+     ocamlfind install FrontC $OCAMLFIND_DESTDIR/frontc/* META
+     rm -rf $OCAMLFIND_DESTDIR/frontc
```

Comment:
If you remove this line, you'll end up with both a /frontc and a /FrontC directory. The few packages which depend on this (not in nixpkgs atm but exist in opam) refer to FrontC, not frontc.

The patch that was previously referenced as `opam_patch` was the patch opam used to accomplish a similar renaming by editing the makefile, but this ended up being easier.